test-jest/polar/RadialBar.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { Surface, RadialBar } from '../../src';
+
+describe('<RadialBar />', () => {
+  const data = [
+    { cx: 250, cy: 250, innerRadius: 0, outerRadius: 0, startAngle: 0, endAngle: 60, value: 50 },
+    { cx: 250, cy: 250, innerRadius: 30, outerRadius: 40, startAngle: 0, endAngle: 60, value: 50 },
+    { cx: 250, cy: 250, innerRadius: 50, outerRadius: 60, startAngle: 0, endAngle: 60, value: 50 },
+    { cx: 250, cy: 250, innerRadius: 70, outerRadius: 80, startAngle: 0, endAngle: 60, value: 50 },
+  ];
+
+  test('Render 5 sectors', () => {
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <RadialBar isAnimationActive={false} cx={250} cy={250} data={data} label dataKey="value" />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.recharts-radial-bar-sector')).toHaveLength(data.length);
+  });
+
+  test("Don't render any sector when data is empty", () => {
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <RadialBar data={[]} dataKey="value" />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.recharts-pie')).toHaveLength(0);
+  });
+
+  test('Render 5 sectors', () => {
+    const onMouseEnter = jest.fn();
+    const onMouseLeave = jest.fn();
+    const onClick = jest.fn();
+
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <RadialBar
+          isAnimationActive={false}
+          cx={250}
+          cy={250}
+          data={data}
+          dataKey="value"
+          onMouseEnter={onMouseEnter}
+          onMouseLeave={onMouseLeave}
+          onClick={onClick}
+        />
+      </Surface>,
+    );
+
+    const sector = container.querySelectorAll('.recharts-sector')[0];
+
+    fireEvent.mouseEnter(sector);
```

Comment:
Any reason to use `fireEvent` over `userEvent`? My understanding is that `userEvent` is the recommended approach

Reply:
I actually didn't know this, nice! 

> userEvent simulates complete interactions, and not only single events

+1 to using `userEvent` where possible

Reply:
@ckifer
Is the one more code PieChart.spec.tsx?

Reply:
I fixed the conflicts, all good now